src/scala/scripts/ScalaPBGenerator.scala

```diff
@@ -39,6 +39,11 @@ class ScalaPBGenerator extends Processor {
       val relativePath = root.relativize(fullPath)
 
       relativePath.toFile.getParentFile.mkdirs
+      // After bazel 0.28 or so, it seems like this shouldn't be required anymore with the virtual imports
+
+      if(relativePath.toFile.exists)
```

Comment:
are you missing `{ }` or something? seems like this is just printing an error message if the file exists.

Reply:
yeah, sorry i wanted to get travis ci to run everything... and have to open a PR to do that. But looks like from the first commit something else blows up for 0.28 before it actually gets to trying out this code.